src/components/simple/alert.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import SbEditable from "storyblok-react";
+import { Alert as DecanterAlert } from "decanter-react";
+import RichTextRenderer from "../../utilities/richTextRenderer";
+import CreateBloks from "../../utilities/createBloks";
+import "../../styles/alert.css";
+
+const Alert = ({ blok }) => {
+  const isLinkDark = blok.type === "warning";
```

Comment:
```suggestion
const Alert = ({ blok: { type: blokType, alertCta, label, heading, hasDismiss, body} }) => {
  const isLinkDark = blokType === "warning";
```

I would prefer that the props of blok be destructured as well.